simulation-model/simulation-model_shacl.ttl

```diff
@@ -0,0 +1,317 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+
+simulation-model:SimulationModelShape a sh:NodeShape ;
+    sh:closed false ;
+    sh:ignoredProperties ( rdf:type ) ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:GeneralShape ;
+            sh:name "general object" ;
+            sh:description "general object with properties for descriptions, data, links, bundle" ;
+            sh:order 1 ;
+            sh:path simulation-model:general ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node simulation-model:FormatShape ;
+            sh:name "format object" ;
+            sh:description "format object with properties for format informations" ;
+            sh:order 2 ;
+            sh:path simulation-model:format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node simulation-model:Content ;
+            sh:name "content object" ;
+            sh:description "content object with properties for simulation-model intrinsic attributes" ;
+            sh:order 3 ;
+            sh:path simulation-model:content ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node simulation-model:QuantityShape ;
+            sh:name "quantity object" ;
+            sh:description "quantity object with properties for quantity informations" ;
+            sh:order 4 ;
+            sh:path simulation-model:quantity ],            
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node simulation-model:QualityShape ;
+            sh:name "quality object" ;
+            sh:description "quality object with properties for quality informations" ;
+            sh:order 5 ;
+            sh:path simulation-model:quality ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node simulation-model:DataSourceShape ;
+            sh:name "data source object" ;
+            sh:description "dataSource object with properties for data sources" ;
+            sh:order 6 ;
+            sh:path simulation-model:dataSource ] ;
+    sh:targetClass simulation-model:SimulationModel .
+ 
+simulation-model:FormatShape a sh:NodeShape ;
+    sh:closed false ;
+    sh:ignoredProperties ( rdf:type ) ;
+    sh:property [ skos:example "OSMP 1.0/OSI 3.1.2/FMI 2.0" ;
+            sh:datatype xsd:string ;
+            sh:description "Format of model implementation, e.g. OSMP 1.0/OSI 3.1.2/FMI 2.0"@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:message "Validation of model.implementation.format failed!"@en ;
+            sh:name "model.implementation.format"@en ;
+            sh:order 0 ;
+            sh:path simulation-model:model.implementation.format ],
```

Comment:
Yes this is a good point. I was also thinking about this. I first stuck to the `.` notation since I remembered a discussion with @pmai regarding the semantically same meaning for attributes and then also keeping the syntax.  At one point in the discussion, we concluded that we want to keep the exact same syntax (this was 1-2 years ago regarding ontology modelling). Thus, it is really clear, that we refer to the same meaning.

I don't have any strong opinion on that one, so @pmai and @robertschubert do you have any opinion? I get the point that this is better for clear meaning but understand, that this cause maybe problems. And for harmonization with the other ontologies this would be also nice. Particularly the hierachy is a clear benfiit now with the hiararchy ability from ontologies/shacle.

And I agree with you, that we shoud split it into `version` and `formatType`. But I'll wait until we get any feedback for the overall naming to apply these changes.